src/suggest-item.jsx

```diff
@@ -19,6 +19,32 @@ export default class SuggestItem extends React.Component {
   }
 
   /**
+   * Checking if item just became active and scrolling if needed.
+   * @param {Object} nextProps The new properties
+   */
+  componentWillReceiveProps(nextProps) {
+    if (nextProps.isActive && !this.props.isActive) {
+      this.scrollIfNeeded();
+    }
+  }
+
+  /**
+   * Scrolling current item to the center of the list if item needs scrolling.
+   * Item is scrolled to the center of the list.
+   */
```

Comment:
You can find demo here: https://mskec.github.io/react-geosuggest/
1. click input
2. use arrow keys to change fixtures, list won't scroll when you go to fixture 6.
3. use mouse scroll to see that there are more fixtures